### PR TITLE
Appbar audit

### PR DESCRIPTION
**Pull request by nucleogenesis**
### Summary

Incrementally applying changes suggested by [this Appbar Audit](https://docs.google.com/document/d/1C7Gj_eQ2FV8adJPpB4PTnd1oJUQxIhpBXbM1aufCWwM/edit#) provided by @khangmach. 

**One significant change** is that when importing from a source, when you drill down into a channel's details, you now see a back arrow and the name of your source (ie, Kolibri Studio, /media/usb, Local Studio (localhost:8080)) - previously, the user would be taken back to the Device>Channels page and would then have to re-select which source to import from.

### Reviewer guidance

1. Please review [this Appbar Audit](https://docs.google.com/document/d/1C7Gj_eQ2FV8adJPpB4PTnd1oJUQxIhpBXbM1aufCWwM/edit#) and see the tables in the lower pages. The left-most column has items describing where in Kolibri the change has been made. I have also linked to screenshots including a Before (on the left) and After (on the right).

### References

[Appbar Audit Document](https://docs.google.com/document/d/1C7Gj_eQ2FV8adJPpB4PTnd1oJUQxIhpBXbM1aufCWwM/edit#)

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by indirectlylit** [comment]
note: milestone is 0.13.0

**Comment by jonboiser** [comment]
In `Coach > Plan > lessons > lesson details page > manage resources > preview resource`, the label is the name of the lesson. It should be the name of the parent topic.

Screen Shot 2019-07-09 at 11 26 03 AM

**Comment by jonboiser** [comment]
When importing from a USB drive, the "Select content from 'channel name' "label still appears briefly before changing to "<- Export to 'drive name'"

Kapture 2019-07-09 at 11 35 56

EDIT from @indirectlylit:  created a new issue for this here: #5759

File: kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateExamPreview.vue [hunk 1]
```diff
@@ -264,6 +264,7 @@
     $trs: {
       title: 'Select questions',
       backLabel: 'Select topics or exercises',
+      appBarLabel: 'Select quiz content',
```
Comment (jonboiser) [inline kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateExamPreview.vue]: The Google Doc says to use "Select exercises" on this label.

**Comment by indirectlylit** [comment]
This looks great. @nucleogenesis I created a new follow-up issue, so if you want to just address the other two items @jonboiser pointed out we can merge this in.

**Comment by nucleogenesis** [comment]
> In `Coach > Plan > lessons > lesson details page > manage resources > preview resource`, the label is the name of the lesson. It should be the name of the parent topic.
> 
> Screen Shot 2019-07-09 at 11 26 03 AM



> This looks great. @nucleogenesis I created a new follow-up issue, so if you want to just address the other two items @jonboiser pointed out we can merge this in.

@indirectlylit @jonboiser I've pushed fixes for two of the items separate from the issue Devon mentioned. Ready for re-review to be sure all looks good.

One thing to note is that when you're managing the resources - if you refresh the page in place, then the `ancestors` data is empty and I cannot find a way to get the name of the parent topic except through that part of the lessonsSummary module.

So - if you go through the tree and get to a preview page, the label will be right - but if you go to that page's URL directly or refresh the page while on it, then it will go to the default "Lesson" lessonLabel text.

**Review by indirectlylit — APPROVED** [review]
thanks @nucleogenesis!

> One thing to note is that when you're managing the resources - if you refresh the page in place, then the ancestors data is empty and I cannot find a way to get the name of the parent topic except through that part of the lessonsSummary module.
> 
> So - if you go through the tree and get to a preview page, the label will be right - but if you go to that page's URL directly or refresh the page while on it, then it will go to the default "Lesson" lessonLabel text.


would you mind opening a new issue for this?

**Pull request merged**